pkgs/development/libraries/hunspell/dictionaries.nix

```
@@ -967,4 +967,33 @@ rec {
       platforms = platforms.all;
     };
   };
+
+  /* ROMANIAN */
+  ro_RO = ro-ro;
+  ro-ro = mkDict rec {
+    pname = "hunspell-dict-ro-ro";
+    version = "3.3.10";
+    shortName = "ro-ro";
+    dictFileName = "ro_RO";
+    fileName = "${dictFileName}.${version}.zip";
+    shortDescription = "Romanian (Romania)";
+    readmeFile = "README";
+
+    src = fetchurl {
+      url = "https://downloads.sourceforge.net/rospell/${fileName}";
+      sha256 = "7f128d64ea06c9e6711c30b118c0afeefb014d8f33c92daccdf455aba2d04519";
```

```suggestion
      hash = "sha256-fxKNZOoGyeZxHDCxGMCv7vsBTY8zyS2szfRVq6LQRRk=";
```

SRI hash is preferred

---

I'm a little busy but I'll solve these two complaints soon

---

Currently having trouble with the SRI hash. How do I generate one more exactly? Just replacing the hash in there with what I get from `sha256sum` does not work.

---

Set it to empty string, rebuild, get the hash from that. Never use `sha256sum` here.

---

Done. Anything else needed? Or any squashes? I got the commit history a bit long